### ISSUE TITLE
Vince cannot mount /data

### ISSUE THREAD

**Issue by ghost** [TeamWin/Team-Win-Recovery-Project#1386]
- [Y] I am running an official build of TWRP, downloaded from https://twrp.me/Devices/
- [Y] I am running the latest version of TWRP
- [Y] I have read the FAQ (https://twrp.me/FAQ/)
- [Y] I have searched for my issue and it does not already exist

**Device codename**: Vince
**TWRP version**: 3.2.3-0

/data is unencrypted

I:Set page: 'mount'
挂载 '/data' 失败 (No such process)
I:Actual block device: '/dev/block/mmcblk0p50', current file system: 'ext4'

[recovery.log](https://github.com/TeamWin/Team-Win-Recovery-Project/files/2789881/recovery.log)
[dmesg.txt](https://github.com/TeamWin/Team-Win-Recovery-Project/files/2789883/dmesg.txt)

**Comment by CaptainThrowback** [comment]
There are no logs. This issue will be closed without more information.

**Comment by ghost** [comment]
> There are no logs. This issue will be closed without more information.

sorry, log attached

**Comment by CaptainThrowback** [comment]
Looks like your data partition is broken, possibly due to an incorrect fstab that doesn't account for the crypto footer. Have you already tried to Format Data in TWRP to start over?

**Comment by ghost** [comment]
> Looks like your data partition is broken, possibly due to an incorrect fstab that doesn't account for the crypto footer. Have you already tried to Format Data in TWRP to start over?

Yes, TWRP is able to mount /data after it has been formated (fastboot -w) , but once the system boot successfully, TWRP will fail to mount /data again

**Comment by eliiik** [comment]
Yes I have the same device (vince) with the same twrp version, and I got the error
```
could not mount data unable to find crypto footer
```
Other devices(Polaris and Whyred) are works well with almost same process. 

I try to format the data partition, and twrp can mount the data partition, too. But once I enter OS and boot back to twrp, it cannot mount the data partition and raise the error.

**Comment by ghost** [comment]
<4>[  100.218954] EXT4-fs warning (device mmcblk0p50): ext4_enable_quotas:5282: Failed to enable quota tracking (type=0, err=-3). Please run e2fsck to fix.
<3>[  100.219194] EXT4-fs (mmcblk0p50): mount failed
